src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/config/routing.yml

```diff
@@ -63,7 +63,10 @@ ss_registration_sms_prove_possession:
 ss_registration_gssf_initiate:
     path:     /registration/gssf/{provider}/initiate
     methods:  [GET]
-    defaults: { _controller: SurfnetStepupSelfServiceSelfServiceBundle:Registration/Gssf:initiate }
+    defaults:
+        _controller: SurfnetStepupSelfServiceSelfServiceBundle:Registration/Gssf:initiate
+        authenticationFailed: false
```

Comment:
Maybe add a doc block for the initiate action in the `GssfController` explaining the defaults that are set for the action and what their intention is.

Reply:
I've documented them in the controller action docblock.